src/cpp/b2w.cpp

```diff
@@ -193,6 +193,8 @@ int main(int argc, char* argv[])
 
         {
             std::vector<int> rLen(lnth, 0);  // list for read start coverage
+            std::vector<char> rd;
+            rd.reserve(301);  // reasonnable initial buffer size to avoid frequent re-allocation. Most HTS tend to use protocols that yield between 100 and 250 bp
```

Comment:
this is fine, most C++ compilers still struggle to understand the reserve semantics and optimize it.

Reply:
`reasonnable` (typo)